receiver/prometheusreceiver/internal/metricsbuilder.go

```diff
@@ -76,11 +88,21 @@ func (b *metricBuilder) AddDataPoint(ls labels.Labels, t int64, v float64) error
 		b.numTimeseries++
 		b.droppedTimeseries++
 		return errMetricNameNotFound
-	} else if shouldSkip(metricName) {
+	} else if isInternalMetric(metricName) {
 		b.hasInternalMetric = true
 		lm := ls.Map()
 		delete(lm, model.MetricNameLabel)
-		b.logger.Debugw("skip internal metric", "name", metricName, "ts", t, "value", v, "labels", lm)
+		switch metricName {
+		case scrapeStatusMetricName:
+			if v == 1.0 {
+				b.scrapeStatus = scrapeStatusOk
+			} else {
+				b.scrapeStatus = scrapeStatusErr
+				b.logger.Infow("http client error:", "ts", t, "value", v, "labels", lm)
```

Comment:
I think that the Prometheus code is the only one that we use the suggared logger, not for this PR but it will be good to use the de-suggared as the rest of the code base.

Reply:
Don't you want this to be warning?

Reply:
Changed the log to a warning.

Reply:
De-sugaring the logger is next on my list of revisions but I wanted that to be in a separate PR.